astrosat/views.py

```diff
@@ -264,17 +267,25 @@ def create_log_records(request):
             request.data, list
         ), "Must supply an array of JSON objects in request"
 
-        for record in request.data:
+        uuids = [uuid.uuid4() for _ in range(len(request.data))]
+
+        for i, record in enumerate(request.data):
             assert "content" in record, "Log Record must contain key 'content' of JSON to be logged"
 
             logger.info(
                 json.dumps(record['content']),
-                extra={"tags": record.get('tags')}
+                extra={
+                    "tags": record.get('tags'),
+                    "uuid": uuids[i]
+                }
             )
     except Exception as ex:
         return Response({"error": str(ex)}, status=status.HTTP_400_BAD_REQUEST)
 
-    return Response({"detail": "Log Created"}, status=status.HTTP_201_CREATED)
+    db_log_records = DatabaseLogRecord.objects.filter(uuid__in=uuids).reverse()
```

Comment:
Don't bother calling `reverse` here.  The order of the output is irrelevant.  If it's causing a test to fail, then order the response content after it return from the view _in the test_.

Reply:
D'oh, makes perfect sense, will do.